test/_init_tests.js

```diff
@@ -265,13 +266,40 @@ sinon.sandbox.create = function(config) {
   return sandbox;
 };
 
+function mockConsoleError() {
```

Comment:
nit: the naming of mockConsoleError and forbidConsoleError bother me, since the "forbidding" part is happening inside the function called "mock". Not sure how I would rename them tho...